addons/paint-skew/addon.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "https://raw.githubusercontent.com/ScratchAddons/manifest-schema/dist/schema.json",
+  "name": "Skewing in costume editor",
```

Comment:
Does this work when you're making a backdrop? If so, this should probably be "Skewing in the paint editor".

Reply:
[The docs](https://scratchaddons.com/docs/reference/addon-manifest/#description) say to use "costume editor" instead of "paint editor".

Reply:
The name "paint editor" comes from 1.4 and isn't used anywhere in the current version of Scratch. We often use "costume" to mean "costume or backdrop", which isn't completely accurate but I don't think it's wrong. Even Scratch does that:
![image](https://user-images.githubusercontent.com/51849865/205600507-8c255c96-034d-4909-82e8-1a7e88466cc5.png)

Reply:
> [The docs](https://scratchaddons.com/docs/reference/addon-manifest/#description) say to use "costume editor" instead of "paint editor".

Oh, thanks

> The name "paint editor" comes from 1.4 and isn't used anywhere in the current version of Scratch. We often use "costume" to mean "costume or backdrop", which isn't completely accurate but I don't think it's wrong. Even Scratch does that:
> ![image](https://user-images.githubusercontent.com/51849865/205600507-8c255c96-034d-4909-82e8-1a7e88466cc5.png)

That's misleading and somewhat confusing, but thanks for pointing that out.